extensions/wikia/PortableInfobox/js/PortableInfobox.js

```diff
@@ -25,22 +25,77 @@
 	};
 
 	var CollapsibleGroup = {
-		init: function($content) {
+		init: function ($content) {
 			var $collapsibleGroups = $content.find('.pi-collapse');
 
-			$collapsibleGroups.each( function( index, group ) {
+			$collapsibleGroups.each(function (index, group) {
 				var $group = $collapsibleGroups.eq(index),
 					$header = $group.find('.pi-header');
 
-				$header.click( function() {
+				$header.click(function () {
 					$group.toggleClass('pi-collapse-closed');
 				});
 			});
 		}
 	};
 
-	mw.hook('wikipage.content').add(function($content) {
+	var CollapsibleData = {
+		verticalPadding: 0,
+		init: function ($content) {
+			var $collapsibleDatas = $content.find(
+					'.portable-infobox > .pi-data-collapse, ' +
+					'.pi-group:not(.pi-collapse-closed) > .pi-data-collapse'
+				),
+				$collapsedGroups = $content.find('.pi-group.pi-collapse-closed');
+
+			if ($collapsibleDatas.length) {
+				// We assume that every collapsible data field has the same paddings
+				// This is being done to avoid multiple DOM queries
+				CollapsibleData.verticalPadding = CollapsibleData.getVerticalPadding($collapsibleDatas.eq(0));
+			}
+
+			$collapsibleDatas.each(CollapsibleData.handleCollapsibleData);
+			$collapsedGroups.each(function () {
+				$(this).on('click', CollapsibleData.onCollapsedGroupClick);
+			});
+		},
+		getVerticalPadding: function ($wrapper) {
+			var $label = $wrapper.find('.pi-data-label'),
+				$value = $wrapper.find('.pi-data-value'),
+				verticalPaddingWrapper = $wrapper.outerHeight() - $wrapper.height(),
```

Comment:
It's not only padding, it contains border size as well. If it's desired then we should think about better name for this variable.